### ISSUE TITLE
Hall of shame 👑

### ISSUE THREAD

**Issue by d2phap** [d2phap/ImageGlass#1413]
![image](https://user-images.githubusercontent.com/3154213/187029452-938a90b8-b9f2-4d6b-81f7-c09010c7cef4.png)

<div align="center">

## To honor the shameful people who violates ImageGlass' copyrights!

</div>

**Comment by d2phap** [comment]
# "HEIC Viewer for Apple Photos" from "Paints Labs"

|     |     |
| --- | --- |
App name | HEIC Viewer for Apple Photos
Author | Paints Labs
MS Store URL | https://microsoft.com/store/productId/9P4VR62PPD1B
Found on | 29/05/2022
My report | https://twitter.com/duongdieuphap/status/1530812541475373056
Status | ✅ Removed


![image](https://user-images.githubusercontent.com/3154213/187030257-825e0c60-a767-4f3a-9068-52654521db5d.png)


![image](https://user-images.githubusercontent.com/3154213/187029839-6e625a23-b7d5-42c0-91e8-4c37192ce8de.png)

![image](https://user-images.githubusercontent.com/3154213/187030251-4d0e8579-2340-4106-891f-274c6947f55f.png)

**Comment by d2phap** [comment]
# "Imageglass Photo Viewer For Windows" from "Aidan Lingin"

|     |     |
| --- | --- |
App name | Imageglass Photo Viewer For Windows
Author | Aidan Lingin
MS Store URL | https://www.microsoft.com/store/productId/9NB81MSN3X0Q
Found on | 27/08/2022
My report | https://twitter.com/duongdieuphap/status/1563491001582632961
Status | ✅ Removed

![image](https://user-images.githubusercontent.com/3154213/187030337-8b753c26-e566-4daf-8af1-d9bd45ff1750.png)

![image](https://user-images.githubusercontent.com/3154213/187030387-b614b8f2-2e27-4f3d-a25c-4b1d56ac18a8.png)


![image](https://user-images.githubusercontent.com/3154213/187030377-d3525b52-cd25-4c70-9bce-cbf8b65cbd07.png)

**Comment by cihe13375** [comment]
Just asking, are they violating ImageGlass's copyrights because they rebrand ImageGlass as something else, or because they don't release their source code? (to be clear, I have no relation with the apps shown above and I didn't use them either)

**Comment by d2phap** [comment]
@cihe13375 yes the brand and they don't publish the source code. And also, when publishing to MS Store, they violate the [policy 11.2](https://docs.microsoft.com/en-us/windows/uwp/publish/store-policies#112-content-including-names-logos-original-and-third-party) which is about the original content: 
> All content in your product and associated metadata must be either originally created by the application provider, appropriately licensed from the third-party rights holder, used as permitted by the rights holder, or used as otherwise permitted by law.

**Comment by d2phap** [comment]
# "Image Viewer - Support All Formats" from "White Moonlight"

|     |     |
| --- | --- |
App name | Image Viewer - Support All Formats
Author | White Moonlight
GitHub profile | https://github.com/moonlight0911 (@moonlight0911) 😂
MS Store URL | https://www.microsoft.com/store/productid/9NHTSWSNTF6V?ocid=pdpshare
Found on | 20/09/2023
My report | https://twitter.com/duongdieuphap/status/1704397964201128225
Status | ✅ Removed


![image](https://github.com/d2phap/ImageGlass/assets/3154213/ee76d552-9d3b-4d58-bb35-bae25ee67ca1)

![image](https://github.com/d2phap/ImageGlass/assets/3154213/934e50ba-edea-4ae4-b19c-1e557e5c3f76)
